src/news.html

```diff
@@ -1,7 +1,7 @@
 ---
 title: Latest News
 layout: default
-permalink: false
+permalink: /news/
```

Comment:
Do we have an initial blog post there? I'd wait on merging until we do

Reply:
Good point! Thank you!

Reply:
i think this is ok for now since we dont link to it anywhere